src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitSessionController.kt

```diff
@@ -180,7 +180,7 @@ class VisitSessionController(
       example = "14:30:00",
     )
     sessionEndTime: LocalTime,
-  ): SessionCapacityDto {
+  ): List<SessionCapacityDto> {
```

Comment:
@tpmcgowan and @hutcheonb-moj this will be interesting because you can now get more that one SessionCapacityDto back... not sure how this is going to work in the front end!

Reply:
interesting question??

Reply:
Simon's responded to say just display a single total capacity - so I guess either merge them into one `SessionCapacityDto` here or we add up multiple ones on the front-end. Probably simpler for you to add and just return one 🤷‍♂️?